client/js/components/drawer.js

```diff
@@ -0,0 +1,25 @@
+class Drawer {
+  constructor(el) {
+    this.el = el;
+    this.trigger = el.getElementsByClassName('js-trigger')[0];
+    this.trigger.addEventListener('click', this.toggleActive.bind(this));
+  }
+
+  getActive() {
+    return this.el.classList.contains('is-active');
+  }
+
+  setActive(value) {
+    if (value) {
+      this.el.classList.add('is-active');
+    } else {
+      this.el.classList.remove('is-active');
+    }
+  }
+
+  toggleActive() {
+    this.setActive(!this.getActive());
+  }
+}
+
+export default Drawer;
```

Comment:
👍  to finding a simpler solution.
Probably better too - as you'd want to see all of them, unless the list get's unwieldy - which maybe means we should start wondering why we have so many components.